tests/trace/test_annotation_feedback.py

```diff
@@ -456,3 +456,57 @@ class FeedbackModel(BaseModel):
     # Invalid cases should return False
     assert not enum_spec.value_is_valid("invalid_choice")
     assert not enum_spec.value_is_valid(123)
+
+
+def test_annotation_feedback_sdk(client):
+    number_spec = AnnotationSpec(
+        name="Number Rating",
+        field_schema={
+            "type": "number",
+            "minimum": 1,
+            "maximum": 5,
+        },
+    )
+    ref = weave.publish(number_spec, "number spec")
+    assert ref
+
+    @weave.op()
+    def do_call():
+        return 3
+
+    do_call()
+    do_call()
+
+    calls = do_call.calls()
+    assert len(list(calls)) == 2
+
+    # Add annotation feedback
+    calls[0].feedback.add(
+        "wandb.annotation.number-spec",
```

Comment:
API wise I don't like that it isn't obvious how to construct this string

Reply:
Yes completely...  We could add a property to the `AnnotationSpec` itself like "feedback_type" that constructs this? @jamie-rasmussen

Reply:
Yes. I guess that also raises the question for me of whether we would want a convenience method on the `AnnotationSpec` that was something like: `number_spec.annotate_call(calls[0], {"value": 3})` - since you have to construct or get the spec anyway?

Reply:
oh interesting... that's a fun idea. Almost like the class is an annotator itself... 

The payload value could be abstracted too...

```python

rating_1_to_10_annotator = AnnotationSpec(...)

rating_1_to_10_annotator.annotate_call(call_id, 3)
```

Reply:
perfect

Reply:
ticketed both things separately